src/interfaces.ts

```diff
@@ -66,6 +66,8 @@ export interface PostGraphileOptions<
   ownerConnectionString?: string;
   // Enable GraphQL websocket transport support for subscriptions (you still need a subscriptions plugin currently)
   subscriptions?: boolean;
+  // ENable GrqphQL websocket transport support for query & Mutation
+  websocketTransport?: boolean;
```

Comment:
i’m not quite happy with this naming but my suggestions aren’t better. Let me ponder.

enableQueriesAndMutationsOverWebsockets
allOperationsOverWebsockets
fullWebsocketSupport
websocketExecute
websocketOperations
graphqlOverWebsocket
wsAllOps